### DISCUSSION TITLE
Crashing with error code -1073740940

### DISCUSSION THREAD

**Discussion by captainkidd5** [amerkoleci/JoltPhysicsSharp#102] [Q&A]
Just a heads up, my project has been constantly crashing after updating to the latest stable version (2.9.13) with error code -1073740940. Not sure what this one means, and unfortunately I don't have a reproduceable project to share atm

**Comment by amerkoleci** [comment]
Hi,
This way I'm not able to help you

**Reply by captainkidd5** [reply]
understandable, just thought I'd put it up here in case the error id had some obvious resolution
